keps/sig-network/2681-pod-host-ip/README.md

```diff
@@ -357,13 +451,18 @@ No
 
 No
 
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+No
+
```

Comment:
Please copy all questions about troubleshooting from the template:
https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template#troubleshooting

even if the answers for them would end up being n/a